### PR TITLE
Remove img requests spam for Logo, AlarmMsg and ServiceMsg on webserver

### PR DESCRIPTION
**Pull request by Steinweber**
### Description

Every second, several requests were sent to the web server to load the same images over and over again. 
The round image next to alarm messages and service messages is reloaded every second. Even if the image does not change. 
Furthermore, the logo is loaded twice every second. These requests are absolutely useless and can be prevented with a few small adjustments.

### Related Issue

None

### Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Alternate Designs

- Replace images by css
- base64 embedded images
 
### Possible Drawbacks

Nothing known

### Verification Process

No function / parameter has been changed. Only the execution of the function. No dependencies available.

### Release Notes

- WebUi optimisation through removal of redundant img-requests

### Contributing checklist

- [x] My code follows the code style of this project.
- [x] I have read the **CONTRIBUTING** and **LICENSE** document.
- [x] I fully agree to distribute my changes under Apache 2.0 license.

### CODE REVIEW

**Comment by jens-maus** [comment]
First of all, thanks for your PR. However, the way you try to integrate it into pur development ecosystem is not correct. Please have a look how we generate/prepare WebUI Patches or provide your proposal as a diff/patch file so that we can replicate your proposed change. So, in the current form I cannot integrate this PR. So please revise!

**Comment by jens-maus** [comment]
@Steinweber I just looked at your PR and reworked it according to our WebUI patchset ecosystem. Please see if all necessary changes are integrated in the final patchset:

https://github.com/jens-maus/RaspberryMatic/blob/d978e0b36d173451a8ad10139d83fe8573d63101/buildroot-external/patches/occu/0153-WebUI-Fix-ImproveImgUpdates.patch#L1-L85

I wil merge this PR now but please reopen this PR or comment on it if you identify changes that you feel are missing. And also please have a look how your changes have been integrated by me now so that you can comply better with a potential new PR you want to submit in future.

**Comment by Steinweber** [comment]
I think I have understood how it works. However, some new questions have arisen in the meantime, which I still have to clarify.
Maybe you have the answers for me.
1. Where do the *.org files come from? Current status of eq-3/occu or the file that is delivered in the current RPMatic version?
2. i run create_patch.sh directly on the RPMatic. However, it has problems with the --label parameter. If I remove it, it works.

Basically:
Do you have any tips on the easiest way to develop (if you have to work on Windows)?
I currently do it like this:
- Load the Git repo into my IDE
- RPMatic installed in VM
- Edit files from VM
- Upload repo in subfolder to create patch

x I haven't found a way to test the patch in a new VM yet.

very cumbersome - but works somehow

**Comment by jens-maus** [comment]
@Steinweber This is not a discussion fora here, but well, I will answer your questions:

> 1. Where do the *.org files come from? Current status of eq-3/occu or the file that is delivered in the current RPMatic version?

It depends where to take the `*.orig` files from. Optimally you would take them from `eq-3/occu` or `jens-maus/occu` directly. However, if the new patch will patch something in the very same line (or surrounding) then you would need to take the latest file from a previous webui patch as the new `*.orig` file.

> 2. i run create_patch.sh directly on the RPMatic. However, it has problems with the --label parameter. If I remove it, it works.

You are not supported to use `create_patches.sh` on RaspberryMatic, but in your build environment which of course have to  have diffutils and a bash shell, yes.

> Basically: Do you have any tips on the easiest way to develop (if you have to work on Windows)?

Install a virtual Linux machine to perform development there and don't use Windows directly. That's the deal!

> I currently do it like this:
> 
> * Load the Git repo into my IDE
> * RPMatic installed in VM
> * Edit files from VM
> * Upload repo in subfolder to create patch

The procedure should be as follows:

1. Create a Linux VM for development (e.g. Ubuntu 20.04)
2. Use ONLY this Linux VM for development
3. Checkout the RaspberryMatic github repo in this Linux-vm 
4. Compile your RaspberryMatic build via `make raspmatic-XXX-release` where `XXX` corresponds to your target platform
5. Install the update and check the changes.

> x I haven't found a way to test the patch in a new VM yet.

You can only do this by re-creating the build. Thus, something like

1. `rm -rf build-raspmatic-XXX/build/occu*`
2. `make raspmatic-XXX-release`

I hope to create some more developer documentation on this matter soon, btw.
